README.md

```diff
@@ -44,7 +44,7 @@ Pkg.add(PackageSpec(url="https://github.com/holgerteichgraeber/ClustForOpt.jl",
 ```
```

Comment:
Above, there is a inconsistency `.jl`. JuMP.jl vs JLD2 for example.

Reply:
Oh right